### DISCUSSION TITLE
confused: id vs group

### DISCUSSION THREAD

**Discussion by anlexN** [directus/directus#15477] [Archive]
![image](https://user-images.githubusercontent.com/8686739/189088289-aff5ddc1-a9de-4214-b83e-4146ee9191c1.png)

**Comment by rijkvanzanten** [comment]
It looks good to me 🤷🏻  Are you running into any particular issues caused by this logic?

**Reply by anlexN** [reply]
so select `id` is just your habit?

**Reply by rijkvanzanten** [reply]
No? In this case, we need to update the fields that are currently in a group, so we select them by their primary key. The primary key of directus_fields is `id`, hence selecting the `id` and using that to update them 👍🏻
